README.md

```diff
@@ -786,34 +770,54 @@ This indicates that there were two files uploaded via the `license_upload` widge
 are `34` and `47`, and can be looked up in the `user_files` table (detailed below) using those ids.
 
 The `user_files` table holds information about a submission's uploaded user files, which includes:
-`file_id`, the corresponding submission's `submission_id`, a `created_at` time,
-the `original_name` of the file, and the S3 `repository_path`.
+`file_id`, the corresponding submission's `submission_id`, a `created_at` time, the `original_name`
+of the file, and the S3 `repository_path`.
 
 ### Deleting Uploaded Files
 
 Upon successful upload a link is provided to delete the file. When the link is used, a browser
 native delete confirmation pop-up will appear asking the user if they are sure they want to delete
-the selected file. If the user selects yes from the pop-up, the file will be deleted
-from the `user_files` table in the database, as well as from S3 storage.
+the selected file. If the user selects yes from the pop-up, the file will be deleted from
+the `user_files` table in the database, as well as from S3 storage.
 
 ### S3 File Retention Policies
 
 We recommend setting a maximum file retention period in S3 by setting up a retention policy on your
-bucket.
-This will automatically delete files in your bucket that are older than the retention policy
+bucket. This will automatically delete files in your bucket that are older than the retention policy
 permits.
 [You can read more about configuring a retention policy in S3 here.](https://docs.aws.amazon.com/AmazonS3/latest/userguide/how-to-set-lifecycle-configuration-intro.html)
 
+### Address Validation
+
+`Form-flow` will support address validation through [Smarty](https://www.smarty.com/). Configuration
+information for smarty can be found [here].
+
+#### Smarty
+
+##### How to configure
+
+Please use the `sample.env` as an example for creating the `.env` a form-flow application. A **
+smarty** auth-token and auth-id must be passed into our `application.yaml` file in order to the
+address validation to work.
+
+```yaml
+  address:
+    smarty:
+      smarty_auth_id: ${SMARTY_AUTH_ID}
+      smarty_auth_token: ${SMARTY_AUTH_TOKEN}
+      smarty_url: "https://us-street.api.smartystreets.com/street-address"
```

Comment:
Given that its under `smarty` already, we could just say `auth_id`, `auth_token`, and `url` instead.

Reply:
Just saw it was [committed like that already](https://github.com/codeforamerica/form-flow-starter-app/pull/66/files) so you can disregard this comment for now.

Reply:
I could see revamping this a little as we go.  

I do wonder if we want the word "validation" in the configuration file to indicate that it is a validation parameter for addresses.  Maybe the configuration could be something like: 

```
  fragments: 
    address:
      validation:
        smarty_auth_id: ${SMARTY_AUTH_ID}
        smarty_auth_token: ${SMARTY_AUTH_TOKEN}
        smarty_url: "https://us-street.api.smartystreets.com/street-address"
```

I'm thinking forward and realize that this is "design as we go" versus before hand. I'm not recommending that we change this now, but maybe as we work through the other tickets it might make sense to change this structure, esp. as we encounter for fragments that might need specific configuration...?

Reply:
Hmmm... above I'm assuming that address validation might be a fragment. It very well may not, so perhaps it will be better to look at this again once we figure out what the specific design is for the fields.